nc/include/Assets.h

```diff
@@ -1,10 +1,74 @@
 #pragma once
 
+#include "physics/Geometry.h"
+
+#include <concepts>
 #include <string>
+#include <vector>
 
 namespace nc
 {
-    void LoadSoundClipAsset(const std::string& path);
-    void LoadConvexHullAsset(const std::string& path);
+    /** Loading assets creates an internal mapping between the asset path and data.
+     *  Duplicate loads are ignored. Objects that use assets do not load them on
+     *  demand, nor do they own the asset data. Assets must be loaded before dependent
+     *  objects are created and should be unloaded only when they are no longer in use. */
+
+    /** Supported file types: .nca */
     void LoadConcaveColliderAsset(const std::string& path);
```

Comment:
What are the rules here around why some methods are named LoadXAsset and some are just LoadX?